lib/project_types/script/layers/infrastructure/languages/assemblyscript_task_runner.rb

```diff
@@ -80,13 +80,10 @@ def check_compilation_dependencies!
           end
 
           def bytecode
-            legacy_filename = format(BYTECODE_FILE, name: script_name)
             filename = format(BYTECODE_FILE, name: "script")
 
             bytecode_file = if ctx.file_exist?(filename)
```

Comment:
we could probably clean up this logic a bit now too, rather than conditionally setting this variable 

```ruby
raise Errors::WebAssemblyBinaryNotFoundError unless ctx.file_exist?(filename)
contents = ctx.binread(filename)
# rest of method ...
```